java/ql/lib/change-notes/2023-09-28-moved-localuserinput-to-mad.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Is this change note needed? From a user's perspective, everything still works the same right? You just moved CodeQL models to models-as-data.

Reply:
The only noticeable difference will be if they add source models for `file`, `environment`, etc. I wasn't sure if it was necessary to add a change note, but I added one to be on the safe side.

I can remove it if we think it's unnecessary.

Reply:
Hm, ok, I see your point. Query developers may be interested in the new source node kinds 👍